provision/vvv-init.sh

```diff
@@ -77,3 +100,9 @@ if [[ ! -d "${VVV_PATH_TO_SITE}/public_html/build" ]]; then
 fi
 
 ln -sf ${VVV_PATH_TO_SITE}/bin/develop_git /home/vagrant/bin/develop_git
```

Comment:
You should probably change this so you can run `develop_git_multisite` so single and multisite can be run side by side.